Kitodo/src/main/java/org/kitodo/export/ExportDms.java

```diff
@@ -70,23 +64,22 @@ public ExportDms(boolean exportImages) {
     }
 
     /**
-     * DMS-Export an eine gewünschte Stelle.
+     * Export to the DMS.
      *
      * @param process
-     *            object
-     * @param destination
-     *            String
+     *            process to export
+     * @param unused
+     *            user home directory
      */
     @Override
-    public boolean startExport(Process process, URI destination) {
-        if (process.getProject().isUseDmsImport()
-                && ConfigCore.getBooleanParameterOrDefaultValue(ParameterCore.ASYNCHRONOUS_AUTOMATIC_EXPORT)) {
-            TaskManager.addTask(new ExportDmsTask(this, process, destination));
+    public boolean startExport(Process process, URI unused) {
```

Comment:
if it is unused, remove it

Reply:
This method overrides a method in a superclass which uses it. Therefore I cannot remove it.